src/conda-packages-solver/index.ts

```diff
@@ -106,22 +121,24 @@ export const initEnv = async (
         const url = item[repoName];
         if (url) {
           const data = await fetchRepodata(url);
-          repodataTotal[repoName] = data;
+          if (data) {
+            repodataTotal[repoName] = data;
+          }
         }
       })
     );
 
     return repodataTotal;
   };
 
-  const fetchRepodata = async (url: string): Promise<Uint8Array> => {
+  const fetchRepodata = async (url: string): Promise<Uint8Array | null> => {
     const options = {
-      headers: { 'Accept-Encoding': 'gzip' }
+      headers: { 'Accept-Encoding': 'zstd' }
     };
 
     const response = await fetch(url, options);
     if (!response.ok) {
-      throw new Error(`HTTP error! status: ${response.status}`);
+      return null;
```

Comment:
Micromamba raises a warning if failing to fetch one of the repodata files, we should probably do the same:

```suggestion
      logger.warn(`Failed to fetch ${url}`)
```